rust/integration-tests/src/sdk_tests/sdk_tests.rs

```diff
@@ -0,0 +1,134 @@
+#[cfg(test)]
+mod tests {
+    use std::fs;
+    use aptos_indexer_processor_sdk::traits::processor_trait::ProcessorTrait;
+    use aptos_indexer_test_transactions::IMPORTED_TESTNET_TXNS_1255836496_V2_FA_METADATA_;
+    use assert_json_diff::assert_json_eq;
+    use diesel::{pg::PgConnection, RunQueryDsl};
+    use sdk_processor::{
+        config::{
+            indexer_processor_config::IndexerProcessorConfig,
+            processor_config::ProcessorConfig,
+        },
+        processors::events::events_processor::{EventsProcessor},
+        schema::events::dsl::*,
+    };
+    use testing_framework::new_test_context::{PostgresTestDatabase, SdkTestContext, TestDatabase};
+    use sdk_processor::config::indexer_processor_config::DbConfig;
+    use diesel::Connection;
+    use crate::diff_test_helper::processors::event_processor::EventsProcessorTestHelper;
+    use crate::diff_test_helper::ProcessorTestHelper;
+    use crate::diff_tests::{remove_inserted_at, remove_transaction_timestamp};
+
+    #[tokio::test]
+    async fn test_run() {
+        // Step 1: set up an input transaction that will be used
+        let imported_txns = [IMPORTED_TESTNET_TXNS_1255836496_V2_FA_METADATA_];
+
+        let txn_version = 1255836496; // TOOD: more elegant way to get this
```

Comment:
I wonder if this can be improved so that they don't have to delcar a txn_version variable with hard-coded value.